Containerfile

```diff
@@ -28,18 +23,18 @@ COPY deploy/ deploy/
 COPY hack/ hack/
 COPY internal/ internal/
 
-RUN make build
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -mod=vendor --ldflags "-s -w" -a -o bin/kubesan cmd/main.go
 
 # CentOS Stream 9 doesn't provide package nbd
 # FROM quay.io/centos/centos:stream9
-FROM quay.io/fedora/fedora:40
+FROM --platform=$TARGETPLATFORM fedora:latest
 
 # util-linux-core, e2fsprogs, and xfsprogs are for Filesystem volume support where
 # blkid(8) and mkfs are required by k8s.io/mount-utils.
-RUN dnf install -qy nbd qemu-img util-linux-core e2fsprogs xfsprogs && dnf clean all
+RUN dnf update -y && dnf install --nodocs --noplugins -qy nbd qemu-img util-linux-core e2fsprogs xfsprogs && dnf clean all
 
-WORKDIR /kubesan
+WORKDIR /
 
-COPY --from=makefile /kubesan/bin/kubesan bin/
+COPY --from=builder /kubesan/bin/kubesan .
 
-ENTRYPOINT [ "/kubesan/bin/kubesan" ]
+ENTRYPOINT [ "/kubesan" ]
```

Comment:
I'm fine with this change but can you share the reason for changing the path?

Reply:
This is more of a nit-picky thing. We usually put the exe in the root but I'm fine leaving it nested

Reply:
I don't care if we un-nest it - as long as it can still be found by the deployment yaml (we are preliminary enough that NOW is the time to sort naming locations, including where important binaries can be found)